factor-bundle cannot be located by Browserify when using common option

When invoking with `common: true`, Browserify is unable to find `factor-bundle` (being a dependency of `atomify-js`, it's amongst its parent's `node_modules` and therefore cannot be resolved):

    Error: Cannot find module 'factor-bundle' from '/path/to/project'
        at Function.module.exports (/path/to/project/node_modules/atomify/node_modules/atomify-js/node_modules/browserify/node_modules/resolve/lib/sync.js:33:11)
        at Browserify.plugin (/path/to/project/node_modules/atomify/node_modules/atomify-js/node_modules/browserify/index.js:291:29)
        at atomifyJs (/path/to/project/node_modules/atomify/node_modules/atomify-js/index.js:224:7)
    ...

    $ find /path/to/project/node_modules -name factor-bundle
    /path/to/project/node_modules/atomify/node_modules/atomify-js/node_modules/.bin/factor-bundle
    /path/to/project/node_modules/atomify/node_modules/atomify-js/node_modules/factor-bundle

--- a/index.js
+++ b/index.js
@@ -20,9 +20,8 @@ var browserify = require('browserify')
   , _ = require('lodash')
   , minifyify = require('minifyify')
   , resrcify = require('resrcify')
+  , factorBundle = require('factor-bundle')
   , ctor
-
-require('factor-bundle')
 
 ctor = module.exports = function atomifyJs (opts, cb) {
   var outputPath
@@ -221,7 +220,7 @@ ctor = module.exports = function atomifyJs (opts, cb) {
     })
 
     // setup factor bundle, pass in our streamable-buffers as the output source
-    b.plugin('factor-bundle', {
+    b.plugin(factorBundle, {
       o: _.values(outputs)
     })
 

